### PR TITLE
update Az.Compute to use latest Compute SDK

### PR DESCRIPTION
**Pull request by grizzlytheodore**
…ges from 'SecurityType" to "string"  end user experience stays the same.

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by grizzlytheodore** [comment]
SecurityType parameter type changed from "SecurityType" to "string". 
because this enum property in Swagger went from modelAsString: false to true. 
shown here https://github.com/Azure/azure-rest-api-specs/commit/9c4f909644f4b0194bd17d02b87c3fd2cd6fb47d

but the end user experience should stay the same.

File: tools/StaticAnalysis/Exceptions/Az.Compute/BreakingChangeIssues.csv [hunk 1]
```diff
@@ -0,0 +1,6 @@
...
+Az.Compute,Microsoft.Azure.Commands.Compute.SetAzureVMSecurityProfile,Set-AzVMSecurityProfile,0,2020,The cmdlet 'Set-AzVMSecurityProfile' no longer supports the type 'Microsoft.Azure.Management.Compute.Models.SecurityTypes' for parameter 'SecurityType'.,Change the type for parameter 'SecurityType' back to 'Microsoft.Azure.Management.Compute.Models.SecurityTypes'.
+Az.Compute,Microsoft.Azure.Commands.Compute.Automation.SetAzureRmVmssSecurityProfileCommand,Set-AzVmssSecurityProfile,0,2020,The cmdlet 'Set-AzVmssSecurityProfile' no longer supports the type 'Microsoft.Azure.Management.Compute.Models.SecurityTypes' for parameter 'SecurityType'.,Change the type for parameter 'SecurityType' back to 'Microsoft.Azure.Management.Compute.Models.SecurityTypes'.
```
Comment (isra-fel) [inline tools/StaticAnalysis/Exceptions/Az.Compute/BreakingChangeIssues.csv:6]: Changing enum to string should not be a breaking change, but removal of properties may cause user's script to break, so they should not be suppressed.

**Review by isra-fel — CHANGES_REQUESTED** [review]
`BreakingChangeIssues.csv` is in wrong format -- each cell should be double-quoted -- make sure you copied them from the csv files in the artifacts of a failed pipeline, not from the logs.

File: tools/StaticAnalysis/Exceptions/Az.Compute/BreakingChangeIssues.csv (hunk 1 above)
Comment (grizzlytheodore) [reply tools/StaticAnalysis/Exceptions/Az.Compute/BreakingChangeIssues.csv:6]: the first two ones are not exactly removed, but 'PublicIpAddressesSkuName' and 'PublicIPAddressSkuTier' are changed to 'name' and 'tier'.    But there are no cmdlets right now that use those models.

**Comment by grizzlytheodore** [comment]
please review again !

File: tools/StaticAnalysis/Exceptions/Az.Compute/BreakingChangeIssues.csv (hunk 1 above)
Comment (isra-fel) [reply tools/StaticAnalysis/Exceptions/Az.Compute/BreakingChangeIssues.csv]: Okay in that case they don't count as breaking change

**Review by isra-fel — APPROVED** [review]

**Pull request merged**